plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/V2Checkpoint.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.deltalake.transactionlog;
+
+import static java.util.Objects.requireNonNull;
+
+public record V2Checkpoint(String path)
+{
+    public V2Checkpoint
```

Comment:
Is it intentional that you don't add the other properties? (taken from one of the resource files of this PR)

```
{
   "path":"00000000000000000001.checkpoint.b6785ba8-b035-4760-b814-1f115ccb6167.parquet",
   "sizeInBytes":23272,
   "modificationTime":1708995923000,
   "nonFileActions":[
      {
         "protocol":{
            "minReaderVersion":3,
            "minWriterVersion":7,
            "readerFeatures":[
               "v2Checkpoint"
            ],
            "writerFeatures":[
               "v2Checkpoint"
            ]
         }
      }
   ]
}
```

Reply:
I don't remember to be honest. But I think we don't need other fields for now.